pkg/qemu/qemu.go

```diff
@@ -600,7 +600,7 @@ func Cmdline(cfg Config) (string, []string, error) {
 		// audio controller
 		args = append(args, "-device", "ich9-intel-hda")
 		// audio codec
-		args = append(args, "-device", fmt.Sprintf("hda-output,audiodev=%s", id))
+		args = append(args, "-device", fmt.Sprintf("hda-duplex,audiodev=%s", id))
```

Comment:
Could you update the comment in `default.yaml`?

Also wondering if we can have a YAML property to disable the audio input

Reply:
@afbjorklund WDYT?

Reply:
A boolean could be added, not sure how to implement it for vz though

Reply:
VZ implementation can be another PR

Reply:
What should we do with this for v0.16.0?

Reply:
My take would be we can support both for now as by default (as we have support for this in both drivers).

Later if we get any usecase for disabling input we will be able to easily provide at that point of time

Reply:
For security I'd prefer to disable input by default

Reply:
Ah yes, i missed the security point. It makes sense.

I will create a PR for vz to support the same.

Reply:
ok, so then we are good for qemu ?